docker-compose.yml

```diff
@@ -829,3 +830,15 @@ services:
       timeout: 10s
       retries: 10
     logging: *logging
+
+  postgresql:
+    image: 'postgres:latest'
```

Comment:
This needs to be based on an environment variable called `POSTGRES_IMAGE`, set up similarly to how we do this for Valkey.